### PR TITLE
Vit9696 contentvisibility 20221225

### PR DESCRIPTION
**Pull request by vit9696**

### CODE REVIEW

File: build_oc.tool [hunk 1]
```diff
@@ -115,27 +115,18 @@ package() {
...
     printf "%s" "OpenCore" > "${dstdir}/${arch}/EFI/OC/.contentFlavour" || exit 1
+    printf "%s" "Disabled" > "${dstdir}/${arch}/EFI/OC/.contentVisibility" || exit 1
 
     local suffix="${arch}"
     if [ "${suffix}" = "X64" ]; then
       suffix="x64"
     fi
     cp "${arch}/Bootstrap.efi" "${dstdir}/${arch}/EFI/BOOT/BOOT${suffix}.efi" || exit 1
     printf "%s" "OpenCore" > "${dstdir}/${arch}/EFI/BOOT/.contentFlavour" || exit 1
+    printf "%s" "Disabled" > "${dstdir}/${arch}/EFI/OC/.contentVisibility" || exit 1
```
Comment (mikebeaton) [inline build_oc.tool:129]: Copy-paste error? I think you meant /BOOT in this second one here.

Comment (vit9696) [reply build_oc.tool]: Nice catch. Fixed, thanks)

File: Library/OcBootManagementLib/BootEntryManagement.c [hunk 2]
```diff
@@ -190,56 +190,117 @@ ExpandShortFormBootPath (
...
+  if (Visibility == NULL) {
+    return BootEntryNormal;
+  }
+
+  if (AsciiStrnCmp (Visibility, "Disabled", L_STR_LEN ("Disabled")) == 0) {
+    return BootEntryDisabled;
+  }
+
+  if (AsciiStrnCmp (Visibility, "Auxiliary", L_STR_LEN ("Auxiliary")) == 0) {
+    return BootEntryAuxiliary;
```
Comment (mhaeuser) [inline Library/OcBootManagementLib/BootEntryManagement.c:296]: These two branches leak ``Visibility``

Comment (vit9696) [reply Library/OcBootManagementLib/BootEntryManagement.c:298]: Fixed, thanks!

**Pull request merged**